### PR TITLE
Problem: Add migrations share approach

### LINKED ISSUE #571: Problem: Add migrations share approach
Currently, migrations need be could copy from this repo to the app. Might need to construct a proper approach to do migration files sharing

### PR DESCRIPTION
**Pull request by davcrypto**
Solution: Resolved #571 by integrated migrations files with projection and implemented script via Golang runtime

# Changes

Migration files will be separated by projections
```
chain-indexing\
  ...
  projection\
    account\
      migrations\
        xxxx.up.sql
        xxxx.down.sql
        xxxxx.up.sql
        xxxxx.down.sql
        ...
        // all migration file related to this projection
    ...
  migrations\
    xxxx.up.sql
    xxxx.down.sql
    // all migration file related to the whole server e.g. 'projections' table
```

1. Separated `xxx_schema_migrations` table will created for each projection e.g. `account_schema_migrations` , `account_message_schema_migrations`
2. Original `schema_migrations` will be used by the outermost migration folder
3. Corresponding `migration.Up()` operation will be forced to execute if the projection is included

### CODE REVIEW

File: projection/bridge_activity/bridge_activity_matcher/bridge_activity_matcher.go [hunk 1]
```diff
@@ -62,6 +66,21 @@ func (cronJob *BridgeActivityMatcher) Config() *Config {
 	return cronJob.Base.Config().(*Config)
 }
 
+const (
+	MIGRATION_TABLE_NAME = "bridge_activity_matcher_migrations"
```
Comment (ysong42) [inline projection/bridge_activity/bridge_activity_matcher/bridge_activity_matcher.go:70]:
```suggestion
	MIGRATION_TABLE_NAME = "bridge_activity_matcher_schema_migrations"
```

File: projection/validatorstats/validatorstats.go [hunk 2]
```diff
@@ -47,7 +62,36 @@ func (_ *ValidatorStats) GetEventsToListen() []string {
 	}
 }
 
+const (
+	MIGRATION_TABLE_NAME = "validator_stats_schema_migrations"
```
Comment (ysong42) [inline projection/validatorstats/validatorstats.go:66]:
```suggestion
	MIGRATION_TABLE_NAME = "validatorstats_schema_migrations"
```

**Review by ysong42 — COMMENTED** [review]
Great job! Overall looks good to me!

Just some minor issues:

- `20211015230145_create_view_bridge_activities.down.sql`
- `20211015230145_create_view_bridge_activities.up.sql` 

should be placed under `projection/bridge_activity/bridge_activity_matcher/migrations`

---
- `20210830173217_view_ibc_channel_messages.down.sql`
- `20210830173217_view_ibc_channel_messages.up.sql`
- `20210910113611_alter_view_ibc_channel_messages_rename_to_view_ibc_channel_traces.down.sql`
- `20210910113611_alter_view_ibc_channel_messages_rename_to_view_ibc_channel_traces.down.sql`

should be placed under `projection/ibc_channel/migrations`

**Comment by davcrypto** [comment]
Thanks @ysong42 Just moved the files to expected position

**Review by ysong42 — APPROVED** [review]
LGTM 👍

**Pull request merged**